tests/core/test_overdrive.py

```diff
@@ -518,9 +518,11 @@ def test_book_info_to_circulation_advantage(self):
         raw, info = self.sample_json("overdrive_availability_advantage.json")
 
         extractor = OverdriveRepresentationExtractor(self.api)
+        # consortial data means data that is available to a consortial account
+        # in other words, the consortial account itself + any shared accounts
```

Comment:
Minor/preference - style/readability: use full sentences, standard capitalization, and punctuation.